recipes/cortecs/meta.yaml

```diff
@@ -7,7 +7,7 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cortecs-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cortecs-0.2.0b0.tar.gz
```

Comment:
You should change this at the top, not here. This will not only confuse folks b/c the conda metadata will 0.2.0 but you will be shipping 0.2.0b, but this also will break the auto-updating bot.

Reply:
@ocefpaf by "at the top" do you mean at the top of this file (i.e. as I set {% set pypi_version = "0.2.0b0" %}) or at PyPI? If the latter I'll wait a few weeks until I push a new non-beta version to PyPI. Thanks for all your help!

Reply:
At the top of the recipe. BTW, by publishing a beta release on conda-forge you should follow:

https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Reply:
@ocefpaf just pushed a non-beta version!